content/docs/user-guide/components/reference/terrain/surface_material_list.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Terrain Surface Material List component
+linktitle: TerrainSurfaceMaterialList
+description: ' Open 3D Engine (O3DE) Terrain Surface Material List reference. '
+weight: 100
+---
+
+The **Terrain Surface Material List** defines mappings between surface types and render materials.  
+You can use this to change the appearance of a surface type within different areas of your game.
+When you assign a material to a surface type, all the visible surfaces of that type, within the bounds of the required
+[Axis Aligned Box Shape](/docs/user-guide/components/reference/shape/axis-aligned-box-shape/), will adopt that material.
+
+
+## Usage ##
+Select the surface type tag using the **SurfaceTag** pull down, then assign a material by using the open button, or by dragging from the AssetBrowser window.
+
+## Provider ##
+
+[Terrain Gem](/docs/user-guide/gems/reference/terrain)
+
```

Comment:
```suggestion

## Dependencies

The [Axis Aligned Box Shape Component](/docs/user-guide/components/reference/shape/axis-aligned-box-shape) is required for the Terrain Surface Materials List component to operate.

```
Suggest adding a dependencies section to match the format of other component reference pages.